doc/modules/computational_performance.rst

```diff
@@ -68,6 +68,23 @@ To benchmark different estimators for your case you can simply change the
 :ref:`sphx_glr_auto_examples_applications_plot_prediction_latency.py`. This should give
 you an estimate of the order of magnitude of the prediction latency.
 
+.. topic:: Configuring Scikit-learn for reduced validation overhead
+
+    Scikit-learn does some validation on data that increases the overhead per
```

Comment:
Because it's discouraged? I feel like public functionality should be documented.... hm...